src/librustdoc/html/static/main.js

```diff
@@ -40,6 +40,29 @@ if (!DOMTokenList.prototype.remove) {
     };
 }
 
+
+// Gets the human-readable string for the virtual-key code of the
+// given KeyboardEvent, ev.
+//
+// This function is meant as a polyfill for KeyboardEvent#key,
+// since it is not supported in Trident.  We also test for
```

Comment:
It turns out Trident is IE 11. But Android doesn't have it either and has a ton more market-share, so I think we should say that instead.
```suggestion
// since it is not supported in IE 11 or Chrome for Android. We also test for
```
https://caniuse.com/?search=KeyboardEvent

Reply:
Oh sure. Just moved the function so didn't look at the docs. Thanks!